docs/monitor-kubernetes.md

```diff
@@ -154,7 +188,7 @@ As a result, you have the `customresource-config-ksm` ConfigMap created.
          -n monitoring-system
         ```
 
-    === "Configuration file" 
+    === ":octicons-file-code-24: Configuration file" 
 
          1. Edit the [`values.yaml`](https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/values.yaml) 
```

Comment:
This section is required only if the user has not changed the values by using --set when installing helm chart.
For example we could achieve the same by using 
```
         --set externalVM.write.url=<PMM-SERVER-URL>/victoriametrics/api/v1/write \
         --set vmagent.spec.externalLabels.k8s_cluster_id=<UNIQUE-CLUSTER-IDENTIFER/NAME>
````
in the helm install command

Reply:
UPdated. Removed config file confoguration section leaving only command line one